test/js-native-api/common.h

```diff
@@ -58,3 +58,24 @@
 
 #define DECLARE_NAPI_GETTER(name, func)                                  \
   { (name), NULL, NULL, (func), NULL, NULL, napi_default, NULL }
+
```

Comment:
There is already 

```
NAPI_EXTERN napi_status
napi_get_last_error_info(napi_env env,
                         const napi_extended_error_info** result);
```


Which includes a more complete erorr string for each error.  I don't think exposing another set of duplicate strings would make sense to me.

Is there some reason that using `napi_get_last_error_info` does not work in this case?